### DISCUSSION TITLE
Where do we see the contents of the default php.ini?

### DISCUSSION THREAD

**Discussion by coopan** [brefphp/bref#1829] [General]
I feel like this I'm missing something obvious as I would think this would be front and centre in the docs since it's the most crucial element of deployed PHP (and a security concern). I cannot seem to find viewable contents for the php.ini file that is put inside `/opt/bref/etc/php` and used by PHP deployed via bref.

Docs have [this page on php.ini](https://bref.sh/docs/environment/php#phpini) that mentions the file, but no contents and no link to it. Searching this repo finds no php.ini file.

It seems the file used to be in [previous versions of the repo](https://github.com/brefphp/bref/blob/1.7.47/runtime/layers/function/php.ini) but searching the latest release for it finds nothing. 

Searching release notes for "ini" yields nothing to explain where it was moved to. And I see no other convos here about it, reaffirming that I've missed something very obvious.

What did I miss? Where is this file coming from now?

**Comment by mnapoli** [comment]
Hi, you can find them here:

- https://github.com/brefphp/aws-lambda-layers/blob/main/layers/fpm/bref.ini
- https://github.com/brefphp/aws-lambda-layers/blob/main/layers/function/bref.ini
